test/MLIR/NativeSpec.hs

```diff
@@ -100,6 +103,24 @@ spec = do
         MLIR.destroyModule m
         str `shouldContain` "loc(\"WhatIamCalled\")"
 
+    it "Can extract first operation (Function) of module" $ \ctx -> do
+      exampleModule <- liftM fromJust $
+        MLIR.withStringRef exampleModuleStr $ MLIR.parseModule ctx
+      operations <- (MLIR.getModuleBody >=> MLIR.getBlockOperations) exampleModule
+      functionStr' <- MLIR.showOperation $ head operations
+      functionStr' `shouldStartWith` "func @add(%arg0: i32) -> i32"
+      MLIR.destroyModule exampleModule
+
+    it "Can show operations inside region of function" $ \ctx -> do
+      exampleModule <- liftM fromJust $
+        MLIR.withStringRef exampleModuleStr $ MLIR.parseModule ctx
+      operations <- (MLIR.getModuleBody >=> MLIR.getBlockOperations) exampleModule
+      blocks <- ((MLIR.getOperationRegion 0) >=> MLIR.getRegionBlocks) (head operations)
+      ops <- MLIR.getBlockOperations $ head blocks
+      opStrs <- sequence $ map MLIR.showOperation ops
+      (BS.intercalate " ; " opStrs) `shouldBe` (pack "%0 = arith.addi %arg0, %arg0 : i32 ; return %0 : i32")
```

Comment:
Do you need this `pack`? I would assume that with overloaded string literals it should work out fine.